bookkeeper-server/src/test/java/org/apache/bookkeeper/client/BookieDecommissionTest.java

```diff
@@ -44,14 +46,17 @@
 
     public BookieDecommissionTest() {
         super(NUM_OF_BOOKIES, 480);
-        baseConf.setOpenLedgerRereplicationGracePeriod(String.valueOf(30000));
-        setAutoRecoveryEnabled(true);
+        baseConf.setOpenLedgerRereplicationGracePeriod(100);
```

Comment:
same comment as above

Reply:
same as above. the point of this change is to reduce the test time. we unnecessarily have a lot of such 30 seconds sleep in quite a few test cases. I would like to kill them or reduce them to a small value.

Reply:
ok..i forgot that in this test testDecommissionForLedgersWithMultipleSegmentsAndNotWriteClosed, there are not writeclosed ledgers and they will remain underreplicated for openLedgerRereplicationGracePeriod. 

Anyhow isn't 100 millsec too less?

Reply:
well I think the test is to wait until the open fragments are fenced (unless I missed something). so if the test is to wait until open fragments are fenced and replicated, then 100ms isn't too less.